### DISCUSSION TITLE
Only simulating 1 out of 20 years

### DISCUSSION THREAD

**Discussion by VimbayiChimonyo** [APSIMInitiative/ApsimX#8795] [General]
### What happened?

[Mbire04_24.zip](https://github.com/APSIMInitiative/ApsimX/files/14664205/Mbire04_24.zip)
 I downloaded weather and soils data from within APSIM and trying to simulate a 20 year virtual experiment with fertiliser and variety. Unfortunately, APSIM is only simulating for 1 year. 

I have attached the simulation for your reference

### What Operating System are you on?

Windows

**Comment by VimbayiChimonyo** [comment]
I am using windows and for the APSIM version it is Version 2023.9.7308.0, built 2023-09-08

**Comment by ric394** [comment]
@VimbayiChimonyo I've run your simulation with '[Clock].EndOfDay' instead of '[Sorghum].Harvesting' in HarvestReport and the simulation runs for the dates set in Clock.
It seems that your only getting harvests in 2008. This is likely to do with your parameters in the SowingRule management script. If you'd like to make sure Sorghum is always sown, you can add the 'else if' statement in the following to your SowingRule:
![image](https://github.com/APSIMInitiative/ApsimX/assets/123442863/ad7d963b-54cc-43df-8b36-9b5a092f5daa)
